File: hospitalManagement/Equipment.cs
using System;
using System.CodeDom;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hospitalManagement
{
    internal class Equipment
    {
        //Field
        private string id;

        // Infor field
        private string name;
        private string description;
        private float price;
        private bool state;
        private string origin;
        private int quantity;

        //  Management field
        // Dynamic field
        // Properties

        public string Id { get => id; set => id = value; }
        public string Name { get => name; set => name = value; }
        public string Description { get => description; set => description = value; }
        public float Price { get => price; set => price = value; }
        public bool State { get => state; set => state = value; }
        public string Origin { get => origin; set => origin = value; }
        public int Quantity { get => quantity; set => quantity = value; }

        // Constructors
        public Equipment() { }
        public Equipment(string id
            , string name
            , string description
            , float price
            , bool state
            , string origin
            , int quantity
            )
        {
            this.id = id;
            this.name = name;
            this.description = description;
            this.price = price;
            this.state = state;
            this.origin = origin;
            this.quantity = quantity;
        }
        // Destructors
        ~Equipment() { }
        // Methods
        // in, output
        public virtual void Input()
        {
            Console.Write("ID: ");
            Id = Console.ReadLine();
            Console.Write("Name: ");
            Name = Console.ReadLine();
            Console.Write("Description: ");
            Description = Console.ReadLine();
            Console.Write("Price: ");
            Price = float.Parse(Console.ReadLine());
            Console.Write("State(1: free, 0: other): ");
            State = Int32.Parse(Console.ReadLine()) == 1 ? true : false;
            Console.Write("Origin: ");
            Origin = Console.ReadLine();
            Console.Write("Quantity: ");
            Quantity = int.Parse(Console.ReadLine());

        }
        public virtual void Output()
        {
            Console.WriteLine($"Id: {Id}");
            Console.WriteLine($"Name: {Name}");
            Console.WriteLine($"Description: {Description}");
            Console.WriteLine($"Price: {Price:C2}");
            Console.WriteLine($"State: {(State ? "Free" : "InWork")}");
            Console.WriteLine($"Origin: {Origin}");
            Console.WriteLine($"Quantity: {Quantity}");
        }


        // General method
        // Other method

        // Overriding
        public override string ToString()
        => $"Id: {Id}"
        + $"\nName: {Name}"
        + $"\nDescription: {Description}"
        + $"\nPrice: {Price}"
        + $"\nState: {State}"
        + $"\nOrigin: {Origin}"
        + $"\nQuantity: {Quantity}";
    }
}

File: src/hospitalManagement/Equipment.java
package hospitalManagement;

import java.util.Scanner;

public class Equipment {
	 //Field
    private String id;

    // Infor field
    private String name;
    private String description;
    private float price;
    private boolean state;
    private String origin;
    private int quantity;

    //  Management field
    // Dynamic field
    // Properties

    

    public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public float getPrice() {
		return price;
	}
	public void setPrice(float price) {
		this.price = price;
	}
	public boolean getState() {
		return state;
	}
	public void setState(boolean state) {
		this.state = state;
	}
	public String getOrigin() {
		return origin;
	}
	public void setOrigin(String origin) {
		this.origin = origin;
	}
	public int getQuantity() {
		return quantity;
	}
	public void setQuantity(int quantity) {
		this.quantity = quantity;
	}
	// Constructors
    public Equipment() { }
    public Equipment(String id
        , String name
        , String description
        , float price
        , boolean state
        , String origin
        , int quantity
        )
    {
        this.id = id;
        this.name = name;
        this.description = description;
        this.price = price;
        this.state = state;
        this.origin = origin;
        this.quantity = quantity;
    }
    // Destructors

    // Methods
    // in, output
    public void Input()
    {
    	Scanner input = new Scanner(System.in);
        System.out.print("ID: ");
        id = input.nextLine();
        System.out.print("Name: ");
        name = input.nextLine();
        System.out.print("Description: ");
        description = input.nextLine();
        System.out.print("Price: ");
        price = input.nextFloat();
        System.out.print("State(1: free, 0: other): ");
        state = input.nextInt() == 1 ? true : false;
        System.out.print("Origin: ");
        origin = input.nextLine();
        System.out.print("Quantity: ");
        quantity = input.nextInt();
        input.close();
    }
    public  void Output()
    {
        System.out.format("Id: %s",id);
        System.out.format("Name: %s",name);
        System.out.format("Description: %s",description);
        System.out.format("Price: %.2f",price);
        System.out.format("State: %s",(state ? "Free" : "InWork"));
        System.out.format("Origin: %s",origin);
        System.out.format("Quantity: %s",quantity);
    }


    // General method
    // Other method

    // Overriding
    public  String toString()
    {return  "Id: "+id
    + "\nName: "+name
    + "\nDescription: "+description
    + "\nPrice: "+price
    + "\nState: "+state
    + "\nOrigin: "+origin
    + "\nQuantity: "+quantity;}
}
